mitodl/mitxpro
# Fixed run_tag data migration

#### Pre-Flight checklist


#### What are the relevant tickets?
No ticket

#### What's this PR do?
Fixes an issue with a data migration introduced in #1573, observed during RC deploy

#### How should this be manually tested?
1. Migrate back to the 0024 courses migration (`manage.py migrate courses 0024`)
1. Check out a commit before the one with the message "Integrated program runs for checkout", such as `e9514bfe0b9037a744c048a634aab621a698b34c`
1. Go into django admin and create a course run with a course and a courseware id suffix that matches another course run. Example: if you have a course run with `courseware_id=course-v1:SEED+AMx+R3`, create a new course run with the same course as that run, and set `courseware_id=course-v1:SEED+AMxCOPY+R3` so the courseware_id's are different but the suffix is the same
1. Check out this branch again
1. Run the migrations

There should be no integrity error and the `run_tag` values should be filled in
